yb-voyager/src/dbzm/config.go

```diff
@@ -80,6 +80,7 @@ type Config struct {
 var baseConfigTemplate = `
 debezium.format.value=connect
 debezium.format.key=connect
+quarkus.http.log=8181
```

Comment:
`quarkus.http.port`

Also, let's use a different port for export data from source vs target